### PR TITLE
Added SignIn Page

### PR DESCRIPTION
**Pull request by SreejaJayan**

### CODE REVIEW

File: app/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -17,6 +17,19 @@
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.2" />
 
+    <com.google.android.material.textfield.TextInputLayout
```
Comment (Harsh2098) [inline app/src/main/res/layout/fragment_login.xml]: Need `TextView` right?

File: app/build.gradle [hunk 2]
```diff
@@ -65,7 +67,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:logging-interceptor:3.12.1'
 
     // Navigation
-    implementation 'android.arch.navigation:navigation-fragment-ktx:1.0.0'
-    implementation 'android.arch.navigation:navigation-ui-ktx:1.0.0'
+    implementation 'android.arch.navigation:navigation-fragment-ktx:1.0.0-alpha11'
```
Comment (Harsh2098) [inline app/build.gradle]: Why change from stable to alpha?

File: app/build.gradle [hunk 3]
```diff
@@ -35,11 +36,12 @@ dependencies {
     implementation 'androidx.preference:preference:1.1.0'
 
     // Kotlin libraries
-    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.2.0'
     implementation "org.jetbrains.anko:anko:0.10.5"
 
     // Testing
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (Harsh2098) [inline app/build.gradle]: Where this was needed?

File: app/src/main/java/com/hmproductions/facewatch/MainActivity.kt [hunk 4]
```diff
@@ -1,11 +1,13 @@
 package com.hmproductions.facewatch
 
 import android.os.Bundle
+import androidx.appcompat.app.ActionBar
```
Comment (Harsh2098) [inline app/src/main/java/com/hmproductions/facewatch/MainActivity.kt]: Unused import?

**Review by Harsh2098 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_login.xml (hunk 1 above)
Comment (SreejaJayan) [reply app/src/main/res/layout/fragment_login.xml]: I'll change that completely and add a dialog box that says new user created.

File: app/build.gradle (hunk 2 above)
Comment (SreejaJayan) [reply app/build.gradle]: I did that coz i thought that will fix an error. But the error was something else. Later I forgot about that.

File: app/build.gradle (hunk 3 above)
Comment (SreejaJayan) [reply app/build.gradle]: I have not idea. I dont even remember adding that. I dont want to remove it coz that might create an error.

File: app/src/main/java/com/hmproductions/facewatch/MainActivity.kt (hunk 4 above)
Comment (SreejaJayan) [reply app/src/main/java/com/hmproductions/facewatch/MainActivity.kt]: Yeah. I'll remove that. Again, dont remember adding that.

File: app/build.gradle (hunk 3 above)
Comment (Harsh2098) [reply app/build.gradle]: What error will it create?

**Review by Harsh2098 — APPROVED** [review]

**Pull request merged**